sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnostics.java

```diff
@@ -42,6 +43,8 @@ public final class CosmosDiagnostics {
     private CosmosDiagnosticsContext diagnosticsContext;
     private final AtomicBoolean diagnosticsCapturedInPagedFlux;
 
+    private AtomicReference<String> trackingId = new AtomicReference<>(null);
```

Comment:
are we planning to add this to diagnostics.toString() as well?

Reply:
Left-over - I have added it to ComsosDiagnosticsContext - which is the right location for something across cross-regional retries

Reply:
Removed the left-overs